app/Http/Controllers/V2/AuditStatus/GetAuditStatusController.php

```diff
@@ -4,18 +4,24 @@
 
 use App\Http\Controllers\Controller;
 use App\Http\Resources\V2\AuditStatusResource;
-use App\Models\V2\EntityModel;
+use App\Models\V2\AuditableModel;
 use Illuminate\Http\Request;
 
 class GetAuditStatusController extends Controller
 {
-    public function __invoke(Request $request, EntityModel $entity)
+    public function __invoke(Request $request, AuditableModel $auditable)
     {
-        $auditStatuses = $entity->auditStatuses()
+        $auditStatuses = $auditable->auditStatuses()
             ->orderBy('updated_at', 'desc')
             ->orderBy('created_at', 'desc')
             ->get();
 
+        $auditStatuses->map(function ($auditStatus) use ($auditable) {
+            $auditStatus->entity_name = $auditable->name;
+
+            return $auditStatus;
+        });
```

Comment:
It's weird to run a `map` function and not do anything with the result. This is probably a bit cleaner as a simple foreach. That also avoids the potential problem of the assignment not working because `$auditStatus` isn't being passed by reference
```
foreach ($auditStatuses as $auditStatus) {
  $auditStatus->entity_name = $auditable->name;
}
```